src/main/java/org/fcrepo/migration/handlers/ocfl/PersistencePaths.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.fcrepo.migration.handlers.ocfl;
+
+/**
+ * This class maps Fedora resources to locations on disk. It is based on this wiki:
+ * https://wiki.lyrasis.org/display/FF/Design+-+Fedora+OCFL+Object+Structure
+ *
+ * This is a simplified implementation because F3 objects are always converted into flat AGs that only contain
+ * binaries and binary descriptions.
+ *
+ * @author pwinckles
+ * @since 6.0.0
+ */
+public final class PersistencePaths {
```

Comment:
Should this class be removed in favor of using: https://github.com/fcrepo4/fcrepo4/pull/1731/files#diff-b7e47c149657e45c2b7a44336a6209f0

Reply:
`migration-utils` doesn't share any dependencies with F6, so it's not possible to use that class here. The version in this project is based on the class in F6, but simplified because all F3 objects are represented as AGs.